### ISSUE TITLE
Missing samples in WC documentation

### ISSUE THREAD

**Issue by nikolam001** [IgniteUI/igniteui-xplat-docs#691]
Some samples from WC documentation point to non-existent files. Either paths are wrong or these files are not yet created.

Full list of missing samples:

- [Getting started page](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/general-getting-started#ignite-ui-web-components-example)
- [Demo inside Selection Types section](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/grids/grid/cell-selection#selection-types)
- [Column Types](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/grids/grid/column-types#web-components-grid-column-types-example)
- [Conditional styling Demo](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/grids/grid/conditional-cell-styling#demo)
- [Demo inside Custom keyboard navigation section](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/grids/grid/keyboard-navigation#custom-keyboard-navigation)
- [Row dragging](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/grids/grid/row-drag#web-components-grid-row-drag-example)
- [Search](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/grids/grid/search#web-components-search-example)
- [Group by](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/grids/grid/groupby#web-components-custom-group-by-example)
- [Remote Oprations](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/grids/pivot-grid/remote-operations)

**Comment by nikolam001** [comment]
FYI @dkamburov

**Comment by HUSSAR-mtrela** [comment]
@nikolam001  @dkamburov first check this [spreadsheet](https://infragisticsinc297-my.sharepoint.com/:x:/g/personal/agoldenbaum_infragistics_com/EWqZZ-O8w2xMjp2EupgeWdQBWg2rbijrda8TaY3pJHjChg?rtime=kceCNx0U20g) to see if those samples were deferred due to issues with the Grid control

**Comment by dkamburov** [comment]
@HUSSAR-mtrela Wasn't there a fallback screen when the sample is missing?
I didn't expected having this message:
![image](https://user-images.githubusercontent.com/1182001/220583259-2f1daca3-5e78-43cc-8da4-fe0e0727c2ea.png)

**Comment by HUSSAR-mtrela** [comment]
yes, there should be a new fallback pages and we just need to push latest samples to production
